packages/core-mobile/app/new/routes/(signedIn)/(modals)/tokenDetail/index.tsx

```diff
@@ -140,6 +141,22 @@ const TokenDetailScreen = (): React.JSX.Element => {
     [selectedSegmentIndex]
   )
 
+  const handleBuy = useCallback(() => {
+    navigate({
+      pathname: '/buy',
+      params: { showAvaxWarning: 'true' }
```

Comment:
```suggestion
      params: { showAvaxWarning: 'false' }
```
this is still part of portfolio so showAvaxWarning is not needed

Reply:
Removed that line